docs/user/phonon/flows/fitting/fitting.md

```diff
@@ -160,24 +163,25 @@ In a similar way, the M3GNet fit hyperparameters can be passed using `make` as w
 
 ```python
 complete_flow = CompleteDFTvsMLBenchmarkWorkflow(
-    ml_models=["M3GNet"], ...,
+    ml_models=["M3GNet"], ..., apply_data_preprocessing=True,
```

Comment:
What preprocessing happens and would the whole fit work without it?

Reply:
Data preprocessing sorts out data points with large forces. It's not implemented for anything else than GAP, but it will also divide the dataset into the different data types (phonon, rattled). Which means that most of the time, nothing happens during this step.

Reply:
But does the workflow run without it?